### DISCUSSION TITLE
Android ver 13 is supported? or any plan?

### DISCUSSION THREAD

**Discussion by todLiu** [DeviceFarmer/stf#689] [Q&A]
Android ver 13 is supported? or any plan?

**Comment by grata-bd** [comment]
It's already supported

**Reply by neha4512** [reply]
OnePlus 12 OxygenOS 14.0 based on Android™ is supported ?? Where i can find the devices list that are supported with Open STF ?

**Reply by jupe** [reply]
I dont think we have any up-to-date list of supported phone models since list would grow every month and some models are available only certain locations. Instead, report if some don’t work and remember to attach all logs (logcat/provider/api,..).. I’ve been used several android versions between 8-15 succefully.

**Reply by neha4512** [reply]
Hi Jupe thanks for replying back. Did you tried with the OnePlus 12 UE with OS 14 ver. Because i am planning to order few of them for my setup its always good to have expert opinion who already worked on that before ordering.

Thanks,
Neha

**Reply by jupe** [reply]
Haven't try exactly that phone&OS. Logcat logs probably tell's what happens under the hoods and why it won't work.
